### ISSUE TITLE
mruby and rubygems

### ISSUE THREAD

**Issue by lewellyn** [rvm/rvm#2610]
It seems that rubygems and mruby are incompatible and that rubygems should not be installed for mruby.

The initial symptom is that rubygems tries to invoke `ruby` with the `-d` option, mruby does not support the option. Removing that option simply cascades to rubygems simply being incompatible (apparently) with mruby (or vice versa).

**Comment by brodock** [comment]
```
rvm reinstall mruby
mruby-1.4.0 - #removing src/mruby-1.4.0..
mruby-1.4.0 - #removing rubies/mruby-1.4.0..
Searching for binary rubies, this might take some time.
No binary rubies available for: osx/10.13/x86_64/mruby-1.4.0.
Continuing with compilation. Please read 'rvm help mount' to get more information on binary rubies.
RVM does not have prediction for required space for mruby-1.4.0, assuming 150MB should be enough, let us know if it was not.
Checking requirements for osx.
Certificates bundle '/usr/local/etc/openssl@1.1/cert.pem' is already up to date.
Requirements installation successful.
Installing Ruby from source to: /Users/brodock/.rvm/rubies/mruby-1.4.0, this may take a while depending on your cpu(s)...
mruby-1.4.0 - #downloading 1.4.0, this may take a while depending on your connection...
mruby-1.4.0 - #extracting 1.4.0 to /Users/brodock/.rvm/src/mruby-1.4.0.....
mruby-1.4.0 - #compiling............................................................
mruby-1.4.0 - #installing.
Install of mruby-1.4.0 - #complete
Making gemset mruby-1.4.0 pristine................................................................................/
'command gem pristine --extensions Usage: --version ruby switches: --version  -b --version load -c --version check -e --version 'command' -v --version print --verbose --version run --version --version print --copyright --version print' failed, you need to fix this gems manually.
.
Error running '__rvm_with mruby-1.4.0 gemset_pristine',
please read /Users/brodock/.rvm/log/1522174698_mruby-1.4.0/gemset.pristine-mruby-1.4.0.log
Making gemset mruby-1.4.0@global pristine................................................................................-
'command gem pristine --extensions Usage: --version ruby switches: --version  -b --version load -c --version check -e --version 'command' -v --version print --verbose --version run --version --version print --copyright --version print' failed, you need to fix this gems manually.
..
Error running '__rvm_with mruby-1.4.0@global gemset_pristine',
please read /Users/brodock/.rvm/log/1522174698_mruby-1.4.0/gemset.pristine-mruby-1.4.0@global.log
```

This seems broken on rvm latest / master.

**Comment by mpapis** [comment]
@brodock this should be fixed on master, if you get any more problems please open a new ticket
